src/pyjion/absint.h

```diff
@@ -330,32 +330,32 @@ class AbstractInterpreter {
     void extendListRecursively(Local list, size_t argCnt);
     void extendList(size_t argCnt);
     void buildSet(size_t argCnt);
-    void unpackEx(size_t size, int opcode);
+    void unpackEx(size_t size, size_t opcode);
 
     void buildMap(size_t argCnt);
 
-    Label getOffsetLabel(int jumpTo);
-    void forIter(int loopIndex);
-    void forIter(int loopIndex, AbstractValueWithSources* iterator);
+    Label getOffsetLabel(size_t jumpTo);
+    void forIter(size_t loopIndex);
+    void forIter(size_t loopIndex, AbstractValueWithSources* iterator);
 
     // Checks to see if we have a null value as the last value on our stack
     // indicating an error, and if so, branches to our current error handler.
-    void errorCheck(const char* reason = nullptr, int curByte = -1);
-    void intErrorCheck(const char* reason = nullptr, int curByte = -1);
+    void errorCheck(const char* reason = nullptr, size_t curByte = ~0);
+    void intErrorCheck(const char* reason = nullptr, size_t curByte = ~0);
```

Comment:
I wasn't sure what to do about these and `branchRaise` below. `~0` seemed to be the most semantically consistent.